corehq/apps/enterprise/templates/enterprise/partials/project_tile.html

```diff
@@ -8,10 +8,20 @@
     <h4 class="fs-5 pt-2">{{ report.title }}</h4>
   </div>
   <div class="card-body pb-1">
-    <div class="js-total fw-bold fs-1">
-      <i class="fa fa-spin fa-spinner"></i>
+    <div class="row justify-content-center">
+      <div class="col-auto align-items-center position-relative">
+        <div class="js-total fw-bold fs-1">
+          <i class="fa fa-spin fa-spinner"></i>
+        </div>
+        {% if report.total_description %}
+          <div class="position-absolute start-100" style="top: 25%">
```

Comment:
The `top: 25%` style could be a `top-25` class, right?

Reply:
Bootstrap only have `top-0`, `top-50` and `top-100`, so I have to use style: https://getbootstrap.com/docs/5.0/utilities/position/#arrange-elements

Reply:
aha, got it

Reply:
can override the sass map to include `top-25`
https://getbootstrap.com/docs/5.3/utilities/position/#sass-maps
also i would prefer `end-0` instead of `start-100`

Reply:
override in 9119c9f35d750088df0342db393871a1ec803986
`start-100` put the left edge of the element at the right edge of the parent element
`end-0` put the right edge of the element at the left edge of the parent element
So they're not exchangeable...